### PR TITLE
Add a few major erotic fiction site and links to a book banned in Malaysia

### PR DESCRIPTION
**Pull request by kaerumy**

### CODE REVIEW

File: lists/global.csv [hunk 1]
```diff
@@ -21,6 +21,7 @@ http://anonymouse.org/,ANON,Anonymization and circumvention tools,2014-04-15,cit
 http://archive.org/,HOST,Hosting and Blogging Platforms,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
 https://archive.org/web/,MMED,Media sharing,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
 https://archiveofourown.org/,CULTR,Culture,2017-10-25,sinarproject,
+https://www.asstr.org/,CULTR,Culture,2019-10-17,sinarproject,erotic fiction
```
Comment (agrabeli) [inline lists/global.csv]: This site doesn't seem to work (perhaps the site is hosted on an unreliable server) -- I'd propose removing it (unless if we can replace it with a different URL?).

File: lists/global.csv [hunk 2]
```diff
@@ -180,6 +182,7 @@ http://mymemory.translated.net/english/,COMT,Communication Tools,2014-04-15,citi
 https://myspace.com/,GRP,Social Networking,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
 http://ncac.org/,NEWS,News Media,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
 https://news.google.com/,NEWS,News Media,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
+http://www.nifty.org/,LGBT,LGBT,2019-10-17,sinarproject,erotic fiction
```
Comment (agrabeli) [inline lists/global.csv]: Please replace this URL with the HTTPS version: https://www.nifty.org/

File: lists/my.csv [hunk 3]
```diff
@@ -22,6 +22,8 @@ http://break-a-leg.blogspot.com/,CULTR,Culture,2016-06-10,OONI,
 https://www.breastcancerfoundation.org.my/,PUBH,Public Health,2018-12-11,sinarproject,
 http://brimas.www1.50megs.com/,ENV,Environment,2014-04-15,citizenlab,
 http://cartoonistsrights.org/designeractivist-fahmy-reza-on-trial-for-online-dissent-in-malaysia/,POLR,Political Criticism,2016-06-10,OONI,
+https://www.cato.org/blog/free-download-banned-malay-edition-islam-without-extremes,REL,Religion,2019-05-06,sinarproject,Post on free download of book "Islam Without Extremes" banned by Home Ministry
+https://object.cato.org/sites/cato.org/files/wp-content/uploads/islam_tanpa_keekstreman_-_2019_revised.pdf,REL,Religion,2019-05-06,sinarproject,Link to free download of book on "Islam Without Extremes" banned by Home Ministry
```
Comment (agrabeli) [inline lists/my.csv]: When accessed from a browser, this seems to redirect to: https://www.cato.org/sites/cato.org/files/wp-content/uploads/islam_tanpa_keekstreman_-_2019_revised.pdf 

Perhaps include this URL?

File: lists/global.csv (hunk 1 above)
Comment (kaerumy) [reply lists/global.csv]: Works for me. Are you using CF 1.1.1.1 DNS by any chance?

Comment (agrabeli) [reply lists/global.csv]: Hm, I'm able to access it now.

**Comment by agrabeli** [comment]
LGTM. Merging this PR.

**Pull request merged**